playbooks/openshift-etcd/private/ca.yml

```diff
@@ -2,6 +2,7 @@
 - name: Generate new etcd CA
   hosts: oo_first_etcd
   roles:
+  - role: openshift_clock
```

Comment:
I was not able to follow why this was added here.

Reply:
This is the first 'etcd' component to be run.  I figured it was a good idea to ensure the clock was correct before running this role (didn't investigate if it's 100% necessary).  Since it's a small role and this play only runs against 1 host, figured it won't hurt.

Would like to eventually put this in prerequisites as I mentioned earlier once that ships.